crates/worker/src/partition/state_machine/tests/kill_cancel.rs

```diff
@@ -105,9 +112,73 @@ async fn kill_inboxed_invocation() -> anyhow::Result<()> {
     Ok(())
 }
 
-#[test(tokio::test)]
-async fn kill_call_tree() -> anyhow::Result<()> {
-    let mut test_env = TestEnv::create().await;
+#[rstest]
+#[case(ExperimentalFeature::InvocationStatusKilled.into(), TerminationFlavor::Kill)]
```

Comment:
Why is ok to leave out the case `#[case(ExperimentalFeature::InvocationStatusKilled.into(), TerminationFlavor::Cancel)]`?

Reply:
It's not really needed, because the `InvocationStatusKilled` affects only `TerminationFlavor::Kill`